test/reaping-faulty-nodes.js

```diff
@@ -218,9 +218,10 @@ test2('tombstone should be gossiped with flag when applied as state', getCluster
             dsl.consumePings(t, tc),
 
             // Wait for a ping from the SUT and validate that it does not gossip about the tombstone
-            dsl.validateEventBody(t, tc, {
-                type: events.Types.Ping,
-                direction: 'request'
+            dsl.validateEventBody(t, tc, function(ping) {
```

Comment:
my suspicion is that the flap happens when the ping sent is to fakeNode 0. This node is the source of the gossip and therefor the tombstone is filtered out in the ping.